### PR TITLE
[mysql] if colunm defined default value and nullable,when column value is null. flink-cdc catch field value is default

### PR DESCRIPTION
**Pull request by GOODBOY008**
fix issue:https://github.com/ververica/flink-cdc-connectors/issues/516

### CODE REVIEW

**Comment by GOODBOY008** [comment]
@leonardBang  The old pr:https://github.com/ververica/flink-cdc-connectors/pull/521 was closed,beacause i delete branch.

**Comment by leonardBang** [comment]
Thanks @GOODBOY008 for the contribution, please check the failed tests

File: flink-connector-mysql-cdc/src/test/resources/ddl/column_type_test.sql [hunk 1]
```diff
@@ -65,6 +65,7 @@ CREATE TABLE full_types (
     multiline_c MULTILINESTRING,
     multipolygon_c MULTIPOLYGON,
     geometrycollection_c GEOMETRYCOLLECTION,
+    nullable_col_with_default_value VARCHAR(255) DEFAULT 'flink',
```
Comment (wuchong) [inline flink-connector-mysql-cdc/src/test/resources/ddl/column_type_test.sql:68]: What about having a short column name after `varchar_c` field? 

`varchar_default_c VARCHAR(255) DEFAULT 'flink'`

**Review by wuchong — COMMENTED** [review]
Thanks for the contribution @GOODBOY008 , I only left a minor comment. 

Besides, could you rebase your branch to latest master?

**Comment by wuchong** [comment]
Hi @GOODBOY008 , the community is planning to release 2.1 soon. If you don't have time to update this PR, we can help to update it.

**Comment by GOODBOY008** [comment]
@wuchong @leonardBang I will update this pr today.

**Comment by wuchong** [comment]
Please do not use "git merge" to rebase branches, otherwise the changes is hard to track. Please use "git rebase" instead. IntelliJ IDEA provide an easy tool to do git rebase, you can find the tool via `VCS -> Git -> Rebase`.

File: flink-connector-mysql-cdc/src/test/java/com/ververica/cdc/connectors/mysql/table/MySqlConnectorITCase.java [hunk 2]
```diff
@@ -1263,6 +1263,65 @@ public void testStartupFromTimestamp() throws Exception {
         result.getJobClient().get().cancel().get();
     }
 
+    @Ignore("https://github.com/ververica/flink-cdc-connectors/issues/516")
```
Comment (wuchong) [inline flink-connector-mysql-cdc/src/test/java/com/ververica/cdc/connectors/mysql/table/MySqlConnectorITCase.java:1266]: Why this is ignored?

Comment (GOODBOY008) [reply flink-connector-mysql-cdc/src/test/java/com/ververica/cdc/connectors/mysql/table/MySqlConnectorITCase.java]: OK,I remove this.

**Review by wuchong — APPROVED** [review]
Thanks for the updating. LGTM.

**Pull request merged**